src/theory/datatypes/inference_manager.h

```diff
@@ -45,16 +44,16 @@ class InferenceManager : public InferenceManagerBuffered
    *
    * @param conc The conclusion of the inference
    * @param exp The explanation of the inference
+   * @param id The inference, used for stats and as a hint for constructing
+   * the proof of (conc => exp)
    * @param forceLemma Whether this inference *must* be processed as a lemma.
    * Otherwise, it may be processed as a fact or lemma based on
    * mustCommunicateFact.
-   * @param i The inference, used for stats and as a hint for constructing
-   * the proof of (conc => exp)
    */
   void addPendingInference(Node conc,
                            Node exp,
-                           bool forceLemma = false,
-                           InferenceId i = InferenceId::UNKNOWN);
+                           InferenceId id,
+                           bool forceLemma = false);
```

Comment:
I think we wanted to put the `id` before `exp`...

Reply:
ah, right.